### ISSUE TITLE
Request: nofollow should be global

### ISSUE THREAD

**Issue by ctrlbrk42** [daledavies/jump#3]
Can you please add nofollow as a global option?

**Comment by daledavies** [comment]
Can't see why not. Do you imagine this would override the site level nofollow settings in sites.json?

**Comment by ctrlbrk42** [comment]
> Can't see why not. Do you imagine this would override the site level nofollow settings in sites.json?

I would make the global a `default` and then the ability to `enable/override` with each individual property.

**Comment by ddproxy** [comment]
Tossing in here - may benefit abstracting the settings to use either YAML or JSON and nesting the sites array for the top-level configuration. Further, can apply configurations to at the root for easier access.

```
default:
  icon: foo.png
  nofollow: true
sites:
  - name: Gitea
    icon: gitea.png
    nofollow: false
```

```
{
  "default": {...},
  "sites": [...]
}
```

**Comment by daledavies** [comment]
Thanks for your suggestions on this @ddproxy, this has been implemented in the latest v1.0.3 release :)
